### DISCUSSION TITLE
Hang/freeze when using git add -p (interactive)

### DISCUSSION THREAD

**Discussion by eppsilon** [microsoft/terminal#18047] [Q&A]
I frequently stage changes in Git repos using `git add -p`. When using it in Windows Terminal, it sometimes will stop responding to all keyboard input. I end up having to close the tab and open a new one. Has anyone else run into this? Is there anything I can do to debug the issue?

**Comment by zadjii-msft** [comment]
Which version of the Terminal are you using? Which version of `git`?

**Reply by eppsilon** [reply]
git 2.46.2.windows.1
PowerShell 7.4.4

**Reply by eppsilon** [reply]
Had it happen again with PS 7.4.6 and git 2.47.0.windows.1. Also, forgot to mention this is on Windows 11 ARM.

**Reply by o-sdn-o** [reply]
Things are:
- Operating system (OS): Win11 on ARM
- Terminal running in the OS: Windows Terminal of unknown version
- Command shell running in the terminal: PowerShell 7.4.6
- Application running in the shell: git 2.47.0.windows.1

**Reply by eppsilon** [reply]
Sorry, the Terminal version is 1.21.2911.0

**Comment by j4james** [comment]
There's a similar issue tracked here: https://github.com/git-for-windows/git/issues/4776
